keyboards/ropro/readme.md

```diff
@@ -0,0 +1,16 @@
+The RoPro
+===
+
+![The_RoPro](https://i.imgur.com/tBubPlp.jpg)
+
+A Compact 75-Key + Rotary Encoder Ortholinear keyboard by [Garret G.](https://github.com/TheRoyalSweatshirt) a.k.a. [/u/The_Royal](https://www.reddit.com/user/The_Royal)  
```

Comment:
You mention a rotary encoder here and the footprint is visible on the PCB image, but there's no code in your PR to manage the rotary encoder.

Reply:
I’m honestly still not 100% Positive on the best way to go about setting up the Rotary Encoder. I’ve read through some documentation but I just thought I should get the bulk of the PR fixed and submitted before working on the Encoder. 

I can work on it tonight if you guys would rather the branch be as complete as possible though.